### PR TITLE
Rebuild page on /certified

### PR DESCRIPTION
**Pull request by minkyngkm**
**BLOCKED:** The`certified/why-certifed` page ([issue](https://github.com/canonical-web-and-design/web-squad/issues/4728)) needs to be merged first 

## Done

- Rebuild page on /certified
- The image in the last section needs to be updated. 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/certified
    - Be sure to test on mobile, tablet and desktop screen sizes
- Please check page against [copy doc](https://docs.google.com/document/d/1Cq9RLNjSbX83WB5fFKYCFRvo_1JEfMydMecqrVeKBP8/edit#) 
- Demo at https://ubuntu-com-11118.demos.haus/certified

## Issue / Card

Fixes [#4697](https://github.com/canonical-web-and-design/web-squad/issues/4697)

## Screenshots

Certified hardware | Ubuntu ()

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

**Comment by davegoddard42** [comment]
LGTM @minkyngkm. Nice work!

File: templates/certified/index.html [hunk 1]
```diff
@@ -41,31 +42,20 @@ <h1>Ubuntu certified hardware</h1>
...
-        loading="auto"
-        ) | safe
-      }}
-      <h2 class="p-card__title p-heading--3">Ubuntu certified desktops</h2>
-      <p class="p-card__content u-sv2">Many of the world's biggest PC manufacturers certify their desktops for Ubuntu, ensuring it always runs as smoothly as its millions of users expect. Ubuntu is fast, reliable, packed with great software and free of viruses. That’s why you’ll find Ubuntu preloaded on desktops across government, education and enterprise, and in homes around the world.</p>
-      <hr />
-      <p class="u-text--muted u-no-margin--bottom">Vendor</p>
+   <div class="col-8">
+      <h2>Ubuntu certified desktops</h2>
+      <p>Many of the world's biggest PC manufacturers certify their desktops for Ubuntu, ensuring it always runs as smoothly as its millions of users expect. Ubuntu is fast, reliable, packed with great software and free of viruses. That’s why you’ll find Ubuntu preloaded on desktops across government, education and enterprise, and in homes around the world.</p>
```
Comment (sowasred2012) [inline templates/certified/index.html:49]: This is different in the copy doc:
```suggestion
      <p>Many of the world's biggest PC manufacturers certify their desktops for Ubuntu, ensuring these systems always run as smoothly as their millions of users expect. Ubuntu is fast, reliable, packed with great software, and free of viruses. That’s why you’ll find Ubuntu preloaded on desktops across government, education, enterprise, and in homes around the world..</p>
```

File: templates/certified/index.html [hunk 2]
```diff
@@ -157,22 +174,15 @@ <h2 class="p-card__title p-heading--3">Ubuntu certified IoT devices</h2>
...
-        ) | safe
-      }}
-      <h2 class="p-card__title p-heading--3">Ubuntu certified SoCs</h2>
-      <p class="p-card__content u-sv2">Low-power small-footprint System on a Chip (SoC) hardware are re-defining the future of sustainable data centers. Ubuntu Core delivers bullet-proof security, reliable updates and access to a rich ecosystem on 32 and 64-bit ARM and X86 platforms, perfect to power the Internet of Things.</p>
+
+  <div class="u-fixed-width u-sv3"><hr></div>
+
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h2>Ubuntu certified SoCs</h2>
```
Comment (sowasred2012) [inline templates/certified/index.html:182]:
```suggestion
      <h2>Ubuntu Certified SoCs</h2>
```

File: templates/certified/index.html [hunk 3]
```diff
@@ -157,22 +174,15 @@ <h2 class="p-card__title p-heading--3">Ubuntu certified IoT devices</h2>
...
+      <p>Low-power small-footprint System on a Chip (SoC) hardware are re-defining the future of sustainable data centers. Ubuntu Core delivers bullet-proof security, reliable updates and access to a rich ecosystem on 32 and 64-bit ARM and X86 platforms, perfect to power the Internet of Things.</p>
```
Comment (sowasred2012) [inline templates/certified/index.html:183]:
```suggestion
      <p>64-bit ARM based System on a Chip (SoC) hardware is re-defining the future of sustainable data centers. Featuring high core counts and lower power consumption and cooling needs, Ubuntu Certified Servers that utilize certified System on Chip hardware can lower your costs while allowing you to scale up and out to meet customer demand.</p>
```

File: templates/certified/index.html [hunk 4]
```diff
@@ -157,22 +174,15 @@ <h2 class="p-card__title p-heading--3">Ubuntu certified IoT devices</h2>
...
       <p>Canonical's certification program offers server manufacturers a selection of SoCs officially supported and maintained in Ubuntu to choose from for their products from servers to IoT.</p>
```
Comment (sowasred2012) [inline templates/certified/index.html:184]:
```suggestion
      <p>Canonical's certification program offers server manufacturers a selection of SoCs officially supported and maintained in Ubuntu to choose from for their products.</p>
```

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
The copy doc has a few big differences, not sure if I'm looking at an old version? https://docs.google.com/document/d/1Cq9RLNjSbX83WB5fFKYCFRvo_1JEfMydMecqrVeKBP8/edit#heading=h.3ziq16wcvhzr

**Comment by davegoddard42** [comment]
Apologies, this is my fault - I should've picked up during review. The copy doc has the correct text - I focused on the visual design when I reviewed the demo webpage, I didn't spot the copy hadn't been updated.

**Comment by minkyngkm** [comment]
@davegoddard42 no worries!  @sowasred2012 I have made some updates. Thanks all :)

**Review by sowasred2012 — APPROVED** [review]
LGTM - thanks!

**Pull request merged**